docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
+visualizations. For visualizations based on saved searches, the search bar is initially disabled. Double-click the 
+grayed-out search field to enable interactive searching.
+
+// Why does it behave this way? I'd like to be able to say 'for saved searches interactive searches are disabled
+// because $REASONS'.
+
+The toolbar at the right of the search box has buttons for creating new visualizations, saving the current 
+visualization, loading an existing visualization, sharing or embedding the visualization, and refreshing the data for 
+the current visualization.
+
+[float]
+[[aggregation-builder]]
+===== Aggregation Builder
+
+Use the aggregation builder on the left of the screen to configure the 
+{ref}/search-aggregations.html#\_metrics_aggregations[metric] and 
+{ref}/search-aggregations.html#\_bucket_aggregations[bucket] aggregations used in your visualization. Buckets are 
+analogous to SQL `GROUP BY` statements. For more information on aggregations, see the main
+{ref}/search-aggregations.html[Elasticsearch documentation].
+
+In bar or line chart visualizations, use _metric_ for the y-axis and _buckets_ are used for the x-axis, segment bar 
+colors, and row/column splits. For pie charts, use the metric for the slice size and the bucket for the number of 
+slices.
+
+// "Other visualizations may use these in new and different ways." < Such as? Would it be useful to add an appendix
+// on advanced visualizations or a cookbook of neat nonintuitive vis tricks?
+
+Choose the metric aggregation for your visualization's Y axis, such as count, average, sum, min, max, or cardinality 
+(unique count). Use bucket aggregations for the visualization's X axis, color slices, and row/column splits. Common 
+bucket aggregations include date histogram, range, terms, filters, and significant terms.
+
+You can set the order in which buckets execute. In Elasticsearch, the first aggregation determines the data set 
+for any subsequent aggregations. The following example involves a date bar chart of Web page hits for the top 5 file 
+extensions.
+
+To use the same extension across all hits, set this order:
+
+1.  *Color:* Terms aggregation of extensions
+2.  *X-Axis:* Date bar chart of `@timestamp`
+
+Elasticsearch collects the records for the top 5 extensions, then creates a date bar chart for each extension.
+
+To chart the top 5 extensions for each hour, use the following order:
+
+1.  *X-Axis:* Date bar chart of `@timestamp` (with 1 hour interval)
+2.  *Color:* Terms aggregation of extensions
+
+For these requests, Elasticsearch creates a date bar chart from all the records, then groups the top five extensions 
+inside each bucket, which in this example is a one-hour interval.
+
+NOTE: Remember, each subsequent bucket slices the data from the previous bucket.
+
+To render the visualization on the _preview canvas_, click the *Apply* button at the bottom of the aggregation builder.
+
```

Comment:
It's hard for me to wrap my head around this in the abstract. I think tying this to some sort of an example would be very useful.